.github/workflows/CI.yml

```
@@ -0,0 +1,114 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }} - Julia ${{ matrix.version }} - ${{ github.event_name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
+          - '1.7'
```

```suggestion
          - '1.6'
```
Julia v1.6 is the LTS. Thus, let's just use that as oldest version to test against.